src/index.ts

```diff
@@ -1,24 +1,51 @@
+import { serve } from "@hono/node-server";
+import { Hono } from "hono";
+import Database from "better-sqlite3";
 
-import express from "express";
-import AppDataSource from "./dataSource"
-import { router } from "./routes/propertyRoutes"
-import { Property } from './entities/Property'
+// =============== //
+// Initialize app  //
+// =============== //
 
-const app = express();
-const PORT = process.env.PORT || 3000;
+// Docs for Hono: https://hono.dev
+const app = new Hono();
```

Comment:
I feel like we should just use express

Reply:
We definitely could, but Hono has some nice out-of-the-box stuff that both Fastify and Express require plugins for.